packages/search-in-workspace/src/browser/search-in-workspace-widget.tsx

```diff
@@ -135,14 +136,20 @@ export class SearchInWorkspaceWidget extends BaseWidget implements StatefulWidge
             enabled: false,
             title: nls.localize('theia/search-in-workspace/includeIgnoredFiles', 'Include Ignored Files')
         };
+        this.symlinkState = {
+            className: codicon('Symlink'),
+            enabled: false,
+            title: nls.localizeByDefault('Follow Symlinks')
+        };
```

Comment:
There is no need for this internal widget state, they are reserved for the different buttons we have in the view (which are used to control the search behavior). We use this state to preserve the options across reloads. Since `followSymlinks` is simply a preference there is no state to preserve.